examples/thermostat/thermostat-common/thermostat.zap

```diff
@@ -9683,6 +9683,66 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
+              "name": "abs min heat setpoint limit",
+              "code": 3,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "1000",
+              "reportable": 0,
+              "minInterval": 0,
+              "maxInterval": 65344,
+              "reportableChange": 0
+            },
+            {
+              "name": "abs max heat setpoint limit",
+              "code": 4,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "2800",
```

Comment:
Likewise, where is this "2800" coming from?  I would have expected "3000".

Reply:
Changed default back to 0x0bb8 in zap